c2mir/c2mir.c

```diff
@@ -4011,7 +4014,12 @@ D (primary_expr) {
   if (MN (T_ID, r) || MN (T_NUMBER, r) || MN (T_CH, r) || MN (T_STR, r)) {
     return r;
   } else if (M ('(')) {
-    P (expr);
+    if (C ('{')) {
```

Comment:
I've used `C` here by looking other code as reference, as there is little explanation on what other matching methods does.